packages/caravan-psbt/src/functions.test.ts

```diff
@@ -0,0 +1,23 @@
+import { bufferize } from "./functions";
+
+describe("bufferize", () => {
```

Comment:
It would also be helpful to have a test when there is an invalid encoding string.

Reply:
[81c17fa](https://github.com/caravan-bitcoin/caravan/pull/91/commits/81c17faab3ef26a5489de1ec8f9cc093248b551a)